src/rules/lowercase-name.ts

```diff
@@ -96,13 +103,18 @@ export default createRule({
             items: { enum: ['describe', 'test', 'it'] },
             additionalItems: false,
           },
+          allow: {
```

Comment:
Name could be clearer IMO, but e.g. `allowedStarts` sound terrible. Can you come up with anything else perhaps?

Reply:
I agree, but really couldn't :joy:

Reply:
Yeah I guess it's slightly better than `allow` IMO.
(Should also rename the variables further at the top of the file then)